update Author TH (#12394)
aspnet/AspNetCore.Docs: aspnetcore/mvc/views/tag-helpers/authoring.md

[!code-csharp[](../../views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/TagHelpers/z1AutoLinkerCopy.cs?highlight=5,6,10&range=8-21)]

* Multiple calls to `GetChildContentAsync` returns the same value and doesn't re-execute the `TagHelper` body unless you pass in a false parameter indicating not to use the cached result.

## Load minified partial view TagHelper

The following code:

> * Inherits from the [Partial Tag Helper](xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper)

Reviewer: ```suggestion
* Inherits from the [Partial Tag Helper](xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper).
```

> * Loads the minified version when the app is not in the development environment.

Reviewer: ```suggestion
* Loads the minified version when the app isn't in the Development environment.
```

> * Loads the minified version when the app is not in the development environment.

Reviewer 2: ```suggestion
* Loads the minified partial view when the app isn't in the Development environment.
```
